### PR TITLE
feat: [OPA-335]: Added Governance landing page icons

### PR DESCRIPTION
**Pull request by stuart-mcdonnell**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/HarnessIcons.ts [hunk 1]
```diff
@@ -339,6 +339,8 @@ import GitopsUnknown from './gitops-unknown.svg'
 import GitopsYes from './gitops-yes.svg'
 import GoogleKubernetesEngine from './google-kubernetes-engine.svg'
 import Google from './google.svg'
+import GovernancePolicyset from './governance-policyset.svg'
```
Comment (Harness-JKhattak) [inline packages/icons/src/HarnessIcons.ts:342]: Is "Policyset" considered one word for the platform or should it be separated?

```suggestion
import GovernancePolicySet from './governance-policy-set.svg'
```

Comment (stuart-mcdonnell) [reply packages/icons/src/HarnessIcons.ts]: Good shout. Updated.

**Review by Harness-JKhattak — DISMISSED** [review]
👍

**Review by knagurski — COMMENTED** [review]
Looks like you can shave off 24.1% of the policyset and 10.8% of the shield if you use an optimiser. I like [ImageOptim](https://imageoptim.com/mac)

**Review by crystal-fong — APPROVED** [review]

**Review by willd0yle — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
🔥

**Review by crystal-fong — APPROVED** [review]

**Pull request merged**